src/hints/mod.rs

```diff
@@ -4,6 +4,9 @@ pub mod execution;
 pub mod syscalls;
 mod unimplemented;
 
+#[cfg(test)]
+mod test;
```

Comment:
I called mine `mod tests` and `tests.rs`. I don't care which we use, I can easily be persuaded to change :)

Reply:
Changed for `tests` and `tests.rs` :)